server/routes/sentry/sentry_routes.py

```
@@ -0,0 +1,486 @@
+
+        logger.info(
+            "[SENTRY] Disconnected user %s (deleted %d token rows)",
+            sanitize(user_id), deleted,
+        )
+        return jsonify({
+            "success": True,
+            "message": "Sentry disconnected successfully",
+            "tokensDeleted": deleted,
+        })
+    except Exception as exc:
+        logger.exception("[SENTRY] Failed to disconnect user %s: %s", user_id, exc)
```

Ruff (0.15.12)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)